Adds gamification: Script to calculate .py code coverage by author.

jamalex opened the pull request:
To ease my guilt from allowing PRs to be merged that reduce test coverage, and to make it easier for us to recover from technical debt once time clears up, I adapted a fun script from:
http://scottlobdell.me/2015/04/gamifying-test-coverage-project/

Our current stats, from master, with special kudos to @ralphiee22 for achieving the apex of test coverage with a golden 100, and a hat-tip to @aronasorman for his judicious pragmatism in the face of deadlines:

```
#1. Rafael Aguayo: 100% coverage (267 out of 267 lines)
#2. Michael Gallaspy: 97% coverage (398 out of 407 lines)
#3. Jamie Alexandre: 94% coverage (2540 out of 2694 lines)
#4. Richard Tibbles: 93% coverage (284 out of 305 lines)
#5. 66eli77: 92% coverage (365 out of 393 lines)
#6. Benjamin Bach: 88% coverage (1743 out of 1961 lines)
#7. Aron Fyodor Asor: 27% coverage (103 out of 372 lines)
```

rtibbles commented:
Worth noting that my stats here are inflated - due to the fact that JS code is only included in coverage if the tests themselves run any of the JS code. So if there are no tests that even cover that code, then it doesn't get included...

rtibbles commented:
Oh, sorry - only .py code... wait, I wrote Python code?

jamalex commented:
> Oh, sorry - only .py code... wait, I wrote Python code?

305 lines of it, apparently!

aronasorman commented:
> Writes script to rank contributors based on coverage
> Assigns bottom ranked person as reviewer

You're playing a dangerous game if you want to get this PR merged ;)

aronasorman commented:
Did you write tests for your script as well? Don't want to see your coverage go down :)

jamalex commented:
> You're playing a dangerous game if you want to get this PR merged ;)

The beautiful thing is that this PR can work its magic without even being merged... :crystal_ball: 

> Did you write tests for your script as well? Don't want to see your coverage go down :)

The other beautiful thing is that code coverage is only applied against the `kolibri` module.

rtibbles commented:
When do we rerun this? :)

jamalex commented:
Sorry for the delay. Updated scoreboard:

```
#1. Rafael Aguayo: 99% coverage (340 out of 341 lines)
#2. Michael Gallaspy: 97% coverage (391 out of 399 lines)
#3. Richard Tibbles: 95% coverage (299 out of 312 lines)
#4. 66eli77: 95% coverage (541 out of 566 lines)
#5. Benjamin Bach: 89% coverage (1748 out of 1960 lines)
#6. Jamie Alexandre: 81% coverage (3082 out of 3767 lines)
#7. Aron Fyodor Asor: 50% coverage (189 out of 378 lines)
```

(@aronasorman's lack of test coverage has infected my score, after I updated some of his code... :cry: )
